app/serializers/v2/move_serializer.rb

```diff
@@ -29,6 +29,9 @@ class MoveSerializer
     has_one :supplier, serializer: SupplierSerializer
 
     has_many :court_hearings, serializer: CourtHearingSerializer
+    has_many :events, serializer: GenericEventSerializer do |object|
```

Comment:
do we not want to add this to v1 as well, as its not a breaking change?

Reply:
No no. V1 is dead and if people want events to be surfaced they can move onto v2